contracts/Staking/Vesting.sol

```diff
@@ -55,7 +57,7 @@ contract Vesting is OwnableUpgradeable {
         emit ScheduleCreated(_receiver, _amount_d18);
     }
 
-    function claim(uint256 from, uint256 to) external {
+    function claim(uint256 from, uint256 to) external claimingNotPaused {
```

Comment:
do we need the modifier on `schedule()`?

Reply:
No, only SRD can call it